apollo-mockserver/gradle.properties

```diff
@@ -1,3 +1,5 @@
 POM_ARTIFACT_ID=apollo-mockserver
 POM_NAME=Apollo Mockserver
 POM_DESCRIPTION=Apollo Mockserver
+# See https://github.com/JetBrains/kotlin/blob/master/kotlin-native/NEW_MM.md
+kotlin.native.binary.memoryModel=experimental
```

Comment:
I'd like to use KGP APIs instead of Gradle properties because:
1. It's easier to find usage and remove them once they become obsolete
2. It allows more fined grained control (see also https://youtrack.jetbrains.com/issue/KT-50552/Gradle-plugin-provide-extension-APIs-instead-of-Gradle-properties)

Here in particular (and in multiple other places), feels like we only need it for the tests. Could we enable it only for the test targets?